Backend/Api/Program.cs

```diff
@@ -20,7 +20,7 @@
 builder.Services.AddSwaggerGen();
 
 
-builder.Services.AddSingleton<IDataRepository, DataRepository>();

```

Comment:
You can test with FakeDataRepositories but rememeber to change back to real DataRepositories before PR, we cannot merge like that